libpius/signer.py

```diff
@@ -497,7 +497,7 @@ def import_clean_key(self, key):
     cmd = [self.gpg] + self.gpg_base_opts + self.gpg_quiet_opts + [
         '--no-default-keyring',
         '--keyring', self.tmp_keyring,
-        '--import-options', 'import-minimal',
+        '--import-options', 'import-minimal,keep-ownertrust',
```

Comment:
This happens for every key on the party keyring which is dangerous.

We probably want something like:

```
import_opts = ['import-minimal']
if key == self.force_signer:
    import_opts.append('keep-ownertrust')
cmd  = ...
    ...
    '--import-options', ','.join(import_opts),
    ...
```